### PR TITLE
Update install.sh

### PR DESCRIPTION
**Pull request by Will-wastelander**
Enable CORS by default

### CODE REVIEW

**Comment by UnchartedBull** [comment]
Thank you for this! Could we maybe have a user prompt before enabling CORS? As this is a change to security in someway it might be good to ask the user whether he wants to enable CORS. It should be clear that OctoDash doesn't work without CORS enabled though.

**Comment by Will-wastelander** [comment]
Testing that I did for latest update.

pi@Ender-3-octo:~ $ /home/pi/oprint/bin/octoprint config get --yaml "api.allowCrossOrigin"
false
...

pi@Ender-3-octo:~ $ bash ncheck_CORS.sh
? Should I enable CORS ? FYI, this is required by OctoDash v3, and may have security implications yes
yes
Enabling CORS ...
pi@Ender-3-octo:~ $ /home/pi/oprint/bin/octoprint config get --yaml "api.allowCrossOrigin"
true
...

**Comment by Will-wastelander** [comment]
image

File: scripts/install.sh [hunk 1]
```diff
@@ -739,6 +739,18 @@ if [ $DIRECTORY != "-" ]; then
   fi;
 fi;
 
+if "$DIRECTORY"/bin/octoprint config get --yaml "api.allowCrossOrigin" | grep -q 'false'; then
+yes_no=( 'yes' 'no' )
+
+list_input ${red}${bold}"Should I enable CORS ? FYI, this is required by OctoDash v3 and OctoPrint 1.6.0, and may have security implications"${normal} yes_no cors
```
Comment (UnchartedBull) [inline scripts/install.sh:745]:
```suggestion
list_input "Should I enable CORS ? FYI, this is required by OctoDash v3 and OctoPrint 1.6.0, and may have security implications"${normal} yes_no cors
```

I would prefer this not being red, as this is nothing bad (yet).

File: scripts/install.sh [hunk 2]
```diff
@@ -739,6 +739,18 @@ if [ $DIRECTORY != "-" ]; then
...
+
+echo $cors
+if [ $cors == 'yes' ]; then
+        echo "Enabling CORS ..."
+        "$DIRECTORY"/bin/octoprint config set --bool "api.allowCrossOrigin" true
+fi
+fi
```
Comment (UnchartedBull) [inline scripts/install.sh:752]:
```suggestion
else
  echo "\e[31mCORS has \e[1mNOT\e[22m been enabled. OctoDash most likely won't work if CORS is disabled. You can always enable it in the API settings in OctoPrint"
fi;
```

and rather print a big warning if the user chooses no.

**Review by UnchartedBull — COMMENTED** [review]
Looks good, I'd rather print the error only if the user selects no though. I haven't tested the proposed changes, but they should work. After this is ready to be merged.

Thanks for all your work! :)

**Comment by Will-wastelander** [comment]
The color codes didn't work. Looks like you want "CORS is NOT" in red, with "NOT" being bold, correct ?

image

**Comment by Will-wastelander** [comment]
This is the best I got. 
image

**Review by UnchartedBull — APPROVED** [review]

**Comment by UnchartedBull** [comment]
Thanks! :)

**Pull request merged**